experimental/framework/fluid-static/src/FluidStatic.ts

```diff
@@ -22,6 +37,31 @@ export class FluidContainer {
         const dataObject = await getObjectWithIdFromContainer<T>(id, this.container);
         return dataObject;
     }
+
+    public getMembers(): IUser[] {
```

Comment:
There's a lot of similarity between this interface and that of the `Audience` itself.  Probably consider just exposing an `IAudience` property to give out the `Audience` itself (i.e. user writes `fluidContainer.audience.getMembers()`).

If we think the `IAudience` interface is not-so-good, then we should probably push the improvements down into it rather than fixing it up with a wrapper.

Reply:
Seconded.

Reply:
This is an interesting concept but I don't know what it looks like in practice. I actually think we need an audience/members class that the Service will bring along. The most interesting part of the Audience is the users object. But this object is defined by the server and specific to the server. The framework is impartial object type.

Reply:
Makes sense, but then until we have deeper thoughts on that service-specific class I think we should probably stay consolidated on `IAudience`.

Another benefit -- this will help ease the transition from consumption-scenario to full Fluid usage if the audience object remains the same across the transition.